pkg/kubelet/cm/topologymanager/topology_manager_test.go

```diff
@@ -86,10 +87,59 @@ func TestNewManager(t *testing.T) {
 				"unknown-option": "true",
 			},
 		},
+		{
```

Comment:
this (welcome) change doesn't seem to be stricly needed for the feature graduation, thus should be moved in its own commit

Reply:
You're right. I will move this to separate commit something along the line `Increase TopologyManager test coverage by adding negative test cases around NUMA topology discovery`

Reply:
I have moved this change into seperate commit.